content/docs/command-reference/add.md

```diff
@@ -41,7 +41,7 @@ Under the hood, a few actions are taken for each file (or directory) in
    <abbr>output</abbr>), using its path and hash to identify the cached data.
    Unless the `-f` option is used, the DVC-file name generated by default is
    `<file>.dvc`, where `<file>` is the file name of the first target.
-5. Unless `dvc init --no-scm` was used when initializing the project, add the
+5. Unless `dvc init --no-git` was used when initializing the project, add the
```

Comment:
Oops please revert changes to actual DVC commands/options like this one because that has to reflect the reality of the DVC core repo/tool and the term SCM is still used there unfortunately. Thanks!

Reply:
Alright.